utilities/zookeeper/R/correctors.county.R

```diff
@@ -79,7 +80,7 @@ zyzzyva_county_corrections <- function(signals_list, ...){
   }
   
   # Multiple Signals
-  if (params$signals_to_correct == "all") {
+  if (params$signals_to_correct == c("deaths_incidence_num","confirmed_incidence_num")) {
     corrected <- list()
     for (i in seq_along(signals_list)) {
       corrected[[i]] <- zyzzyva_county_corrections_single_signal(
```

Comment:
Two concerns here. `params$signals_to_correct` will have to be exactly `c("deaths_incidence_num","confirmed_incidence_num")`. It can't be `c("confirmed_incidence_num","deaths_incidence_num")`. Either should be ok. And looking at line 107 below, it appears that only `confirmed_incidence_num` will ever get corrected.